src/planet-4-151612/wordpress/wp-config.php.tmpl

```diff
@@ -93,11 +93,13 @@ define( 'WP_REDIS_DEFAULT_EXPIRE_SECONDS',   '{{ .Env.WP_REDIS_DEFAULT_EXPIRE_SE
 {{ end }}
 {{ end }}
 
-/**
- * Smartsheet credentials.
- */
 define('SMARTSHEET_API_KEY', '{{ .Env.SMARTSHEET_API_KEY }}');
 
+{{ if .Env.GOOGLE_SHEETS_KEY }}
+define( 'GOOGLE_SHEETS_KEY',   '{{ .Env.GOOGLE_SHEETS_KEY }}' );
+{{ end }}
+{{ end }}
```

Comment:
Why the double `{{ end }}` here?

Reply:
Typo because I copied the section above it.